packages/slice-machine/src/legacy/lib/builders/common/EditModal/index.jsx

```diff
@@ -234,6 +234,8 @@ const EditModal = ({ close, data, fields, onSave, zoneType }) => {
                   />
                   <Close // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment
                     onClick={close}
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Let's add this comment at the top of the file so we don't have to add it in each line.

Reply:
forgot to comment that it was done, sorry